tests/unit/test_charm.py

```diff
@@ -41,6 +41,7 @@ def test_install_event(update, harness):
     harness.set_leader(True)
     harness.begin()
 
+    harness.set_can_connect("dex", True)
```

Comment:
Is this caused by us bumping `ops` to 2.4.1?

Reply:
Reverted.